src/main/java/seedu/address/model/person/PersonFieldsContainKeywordPredicate.java

```diff
@@ -39,7 +39,9 @@ public PersonFieldsContainKeywordPredicate(Map<String, String> categoryDescripti
      */
     @Override
     public boolean test(Person person) {
+        assert categoryDescriptionMap != null;
```

Comment:
Nice job on adding the assertions to make sure the categoryDescriptionMap is not null.